scanners/git-repo-scanner/scanner/git_repo_scanner/__main__.py

```diff
@@ -117,54 +123,63 @@ def get_parser_args(args=None):
         required=True,
     )
     parser.add_argument(
-        "--file-output", help="The path of the output file", required=True
+        "--file-output", 
+        help="The path of the output file", 
+        required=True
```

Comment:
In addition to the check above, we might want to add a validation using the `type` argument.

For example, add the argument here:
`type=is_or_can_be_a_file`

and define the function:
```python
def is_or_can_be_a_file(value: str) -> Path:
    """Checks whether the given string is a path to an existing or non-existing file and if so, returns it as Path.

    Args:
        value: The string to process.

    Returns:
        The corresponding Path instance.
    """
    path = Path(value)
    if path.exists() and not path.is_file():
        raise argparse.ArgumentTypeError(f"{value} is not a file or cannot be created as such")
    return path.absolute()
```

Notice: It does not handle non-existing parent directories.